alchemiscale/interface/client.py

```diff
@@ -243,6 +257,25 @@ def set_network_weight(self, network: ScopedKey, weight: float) -> None:
         """
         self._post_resource(f"/networks/{network}/weight", weight)
 
+    async def _set_network_weight(self, networks: List[ScopedKey], weight) -> None:
+        data = dict(networks=[str(network) for network in networks], weight=weight)
+        await self._post_resource_async("/bulk/networks/weight/set", data=data)
+
+    def set_networks_weight(
+        self,
+        networks: List[ScopedKey],
+        weight: float,
```

Comment:
This might be challenging to do, but I think supporting `weight` as a list of floats equal in length to `networks` is desirable, as this would allow users to use `get_networks_weight` to get the weights for many networks, alter them, then use `set_networks_weight` with these altered weights to change them in one call.

What do you think @ianmkenney?